src/ceph_fuse.cc

```diff
@@ -312,7 +312,9 @@ int main(int argc, const char **argv, const char *envp[]) {
 
     r = cfuse->start();
     if (r != 0) {
-      cerr << "ceph-fuse[" << getpid() << "]: fuse failed to start" << std::endl;
+      if (r != EBUSY) {
+	cerr << "ceph-fuse[" << getpid() << "]: fuse failed to start" << std::endl;
+      }
```

Comment:
since we are going to return an error code for `EBUSY`, why wouldn't we output this message for `EBUSY`?

Reply:
Keeping message for EBUSY too